mne/viz/tests/test_ica.py

```diff
@@ -73,6 +73,7 @@ def test_plot_ica_sources():
     """
     import matplotlib.pyplot as plt
     raw = io.Raw(raw_fname, preload=True)
+    raw.crop(0, 1, copy=False)
```

Comment:
should be faster to have `preload=False` above, then crop, then say `raw.preload_data()`. A small difference I know but our test suite is already slow.